src/GraphQL/Types/Scalars/TimeOnlyGraphType.cs

```diff
@@ -0,0 +1,61 @@
+#if NET6_0_OR_GREATER
+
+using System;
+using System.Globalization;
+using GraphQL.Language.AST;
+
+namespace GraphQL.Types
+{
+    /// <summary>
+    /// The Time scalar graph type represents a time in accordance with the ISO-8601 standard.
+    /// Format is `HH:mm:ss.fffffff`.
+    /// </summary>
+    public class TimeOnlyGraphType : ScalarGraphType
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="TimeOnlyGraphType"/> class.
+        /// </summary>
+        public TimeOnlyGraphType()
+        {
+            Description = "The `Time` scalar type represents a time in accordance with the " +
+                "[ISO-8601](https://en.wikipedia.org/wiki/ISO_8601) standard. Format is `HH:mm:ss.fffffff`.";
```

Comment:
```suggestion
                "[ISO-8601](https://en.wikipedia.org/wiki/ISO_8601) standard. Format is `HH:mm:ss.FFFFFFF`.";
```
I missed this one. For consistency let’s change it too. Otherwise looks good.